### ISSUE TITLE
unoptimized images are not following basepath

### ISSUE THREAD

**Issue by chandrahaas02** [vercel/next.js#68498]
### Link to the code that reproduces this issue

https://github.com/chandrahaas02/portfolio

### To Reproduce

by adding unoptimized : true  we are also skipping basepath formatting to image src's srcs

### Current vs. Expected behavior

It should add basepath to even it is unoptimized

### Provide environment information

```bash
Operating System:
  Platform: linux
  Arch: x64
  Version: #1 SMP PREEMPT_DYNAMIC Thu Jul 25 18:17:34 UTC 2024
  Available memory (MB): 15690
  Available CPU cores: 12
Binaries:
  Node: 18.20.2
  npm: 10.5.0
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 14.2.5 // Latest available version is detected (14.2.5).
  eslint-config-next: 14.2.5
  react: 18.3.1
  react-dom: 18.3.1
  typescript: 5.5.4
Next.js Config:
  output: export
```

### Which area(s) are affected? (Select all that apply)

Image (next/image)

### Which stage(s) are affected? (Select all that apply)

next build (local)

**Comment by ugursepetci** [comment]
doesn't it work that way?

<img width="193" alt="Screenshot 2024-08-04 at 22 08 13" src="https://github.com/user-attachments/assets/5c6c1f91-000c-4f14-b022-9d4ed8e997c0">

**Comment by chandrahaas02** [comment]
hey @ugursepetci yeah I tried that also but it is not adding basePath  so github actions for some reason want images unoptimized because this github pages build is not requesting the correct image

![exported index.html](https://github.com/user-attachments/assets/b04b8b7b-b01f-4f05-88ab-1d89a8240ca4)

**Comment by samcx** [comment]
@chandrahaas02 You're referring to `basePath` in your `next.config.mjs`, correct (your <picture data-single-emoji=":repro:" title=":repro:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/repro/5cea53b5532825b4.png" alt=":repro:" align="absmiddle"></picture> is missing this)?

**Comment by chandrahaas02** [comment]
> @chandrahaas02 You're referring to `basePath` in your `next.config.mjs`, correct 

yeah I just avoided it there since github automatically adds that but yeah I just removed that and added it back here so that it is better visible 

you can also find an open github issue  https://github.com/actions/configure-pages/issues/67 which is very similar

**Comment by samcx** [comment]
@chandrahaas02 You have to statically import it and move the image our of `/public`.

```
import haas from "./haas.png"

<Image src={haas} width={500} height={500} alt="me" />
```
